src/User/Controller/AuthController.php

```diff
@@ -107,12 +107,11 @@ public function register(Request $request, Response $response): Response
         $parsedData = $request->getParsedBody();
 
         $this->validationService->validate($parsedData, [
-            UserInterface::COLUMN_USERNAME => 'required|min:2|max:12|regex:/^[a-zA-Z\d]+$/',
+            UserInterface::COLUMN_USERNAME => 'required|min:3|max:12|regex:/^[a-zA-Z]+([\.:_\-=!\?]){0,5}([a-zA-Z0-9])+$/',
             UserInterface::COLUMN_MAIL => 'required|email|min:9',
             UserInterface::COLUMN_LOOK => 'required',
             UserInterface::COLUMN_GENDER => 'required|default:M|regex:/[M.F]/',
-            UserInterface::COLUMN_PASSWORD => 'required|min:6',
-            'password_confirmation' => 'required|same:password'
+            UserInterface::COLUMN_PASSWORD => 'required|min:6','password_confirmation' => 'required|same:password'

```

Comment:
What's the reason for this style change ?

Reply:
For some reason, I changed many lines same way, I found it better to read and if lines are too large on VS Code I just go _Alt + Z_